bitbots_motion/bitbots_head_mover/src/move_head.cpp

```diff
@@ -56,7 +56,7 @@ class HeadMover {
   std::shared_ptr<tf2_ros::TransformListener> tf_listener_;
 
   // Declare variables
-  uint head_mode_;
+  uint head_mode_ = 7;
```

Comment:
Please avoid magic values.
```suggestion
  uint head_mode_ = humanoid_league_msgs::msg::HeadMode::LOOK_FORWARD;
```

Reply:
Oh yeah this was by accident. I totally agree with you.